optimum/habana/accelerate/utils/transformer_engine.py

```diff
@@ -75,6 +77,10 @@ def _convert_model(model, to_transformer_engine=True, _convert_linear=True):
                 new_module.bias.copy_(module.bias)
 
             setattr(model, name, new_module)
+        elif isinstance(module, ModuleFusedSDPA) and to_transformer_engine:
+            from habana_frameworks.torch.hpex.experimental.transformer_engine import FusedAttention as te_FusedAttention
+            module._hpu_kernel_fsdpa = te_FusedAttention(scale=module.scale, attention_dropout=module.attention_dropout, enable_recompute=False)
```

Comment:
Should we have enable_recompute = module.enable_recompute?

Reply:
Done